templates/blog/post_list.html

```diff
@@ -0,0 +1,23 @@
+{% extends 'base.html' %}
+
+{% block title %}
+  <title>Blog Posts</title>
+{% endblock %}
+
+{% block content %}
+  <h2>Posts</h2>
+  <div>
+    <ul>
+      {% for post in post_list %}
+        <li>
+          <h3><a href="{% url 'blog:post-detail' pk=post.id %}">{{ post.title }}</a></h3>
+          <div> Author: {{ post.owner }}</div>
```

Comment:
You are displaying the author as `post.owner`. If the `owner` is a User object, this will display the user's `str()` representation, which, depending on your User model, might not be the author's name. Ensure that you are displaying the correct, readable information about the author.